cores/esp8266/Schedule.cpp

```diff
@@ -117,8 +117,6 @@ bool schedule_recurrent_function_us(const std::function<bool(void)>& fn,
 
 void run_scheduled_functions()
 {
-    esp8266::polledTimeout::periodicFastMs yieldNow(100); // yield every 100ms
```

Comment:
Can this be separated out into a separate PR?  Seems like changes in Schedule.cpp are not directly related to the optimistic_yield fix.

Reply:
This Github review presentation is useless, I'll have to browse the context to refresh my memory :-) So: I'll comment later.

Reply:
You're right, this was giving myself an unnecessary merge conflict. I've removed this.